.travis.yml

```diff
@@ -38,68 +39,44 @@ script:
 # - bundle install
 # - bundle exec rails webpacker:install
 # - bundle exec rails webpacker:compile
-
-deploy:
-- provider: elasticbeanstalk
-  skip_cleanup: true
-  access_key_id: "$AWS_ACCESS_KEY_ID_DEVELOPMENT"
-  secret_access_key: "$AWS_SECRET_ACCESS_KEY_DEVELOPMENT"
-  region: us-east-1
-  app: my-library-nyc-app
-  env: my-library-nyc-app-react-dev-18
-  bucket_name: elasticbeanstalk-us-east-1-224280085904
-  bucket_path: my-library-nyc-app-react-dev-18
-  on:
-    repo: NYPL/MyLibraryNYCApp
-    branch: development
-- provider: elasticbeanstalk
-  skip_cleanup: true
-  access_key_id: "$AWS_ACCESS_KEY_ID_QA"
-  secret_access_key: "$AWS_SECRET_ACCESS_KEY_QA"
-  region: us-east-1
-  app: my-library-nyc-app
-  env: my-library-nyc-app-qa25
-  bucket_name: elasticbeanstalk-us-east-1-946183545209
-  bucket_path: my-library-nyc-app-qa25
-  on:
-    repo: NYPL/MyLibraryNYCApp
-    branch: qa-old
-- provider: elasticbeanstalk
-  skip_cleanup: true
-  access_key_id: "$AWS_ACCESS_KEY_ID_QA"
-  secret_access_key: "$AWS_SECRET_ACCESS_KEY_QA"
-  region: us-east-1
-  app: my-library-nyc-app
-  env: my-library-nyc-app-react-qa-27
-  bucket_name: elasticbeanstalk-us-east-1-946183545209
-  bucket_path: my-library-nyc-app-react-qa-27
-  on:
-    repo: NYPL/MyLibraryNYCApp
-    branch: qa
-- provider: elasticbeanstalk
-  skip_cleanup: true
-  access_key_id: "$AWS_ACCESS_KEY_ID_PRODUCTION"
-  secret_access_key: "$AWS_SECRET_ACCESS_KEY_PRODUCTION"
-  region: us-east-1
-  app: my-library-nyc-app
-  env: my-library-nyc-app-production
-  bucket_name: elasticbeanstalk-us-east-1-946183545209
-  bucket_path: my-library-nyc-app-production
-  on:
-    repo: NYPL/MyLibraryNYCApp
-    branch: master
-- provider: elasticbeanstalk
-  skip_cleanup: true
-  access_key_id: "$AWS_ACCESS_KEY_ID_PRODUCTION"
-  secret_access_key: "$AWS_SECRET_ACCESS_KEY_PRODUCTION"
-  region: us-east-1
-  app: my-library-nyc-app
-  env: my-library-nyc-app-react-production
-  bucket_name: elasticbeanstalk-us-east-1-946183545209
-  bucket_path: my-library-nyc-app-react-production
-  on:
-    repo: NYPL/MyLibraryNYCApp
-    branch: production
+- stage: deploy qa
```

Comment:
Does the development branch deployment still need space here? A development ECS stack could be created on the `nypl-sandbox` account which is the development account for LSP, and the `nypl-digital-dev` account acts as the qa and production account.

Reply:
@gkallenberg DIscussed with Stephen today we don't need Development server for MylibraryNyc. Here no need to give space.